cmake/macros/TargetOpenVR.cmake

```
@@ -6,6 +6,13 @@
 #  See the accompanying file LICENSE or http://www.apache.org/licenses/LICENSE-2.0.html
 #
 macro(TARGET_OPENVR)
+  if(OVERTE_USE_SYSTEM_LIBS)
+    find_package(PkgConfig REQUIRED)
+    pkg_check_modules(OpenVR REQUIRED openvr)
+    target_include_directories(${TARGET_NAME} SYSTEM PRIVATE ${OpenVR_INCLUDE_DIRS})
+    target_link_libraries(${TARGET_NAME} ${OpenVR_LINK_LIBRARIES})
+  else()
     find_package(OpenVR REQUIRED)
     target_link_libraries(${TARGET_NAME} openvr::openvr)
+  endif()
```

```suggestion
    if(OVERTE_USE_SYSTEM_LIBS)
        find_package(PkgConfig REQUIRED)
        pkg_check_modules(OpenVR REQUIRED openvr)
        target_include_directories(${TARGET_NAME} SYSTEM PRIVATE ${OpenVR_INCLUDE_DIRS})
        target_link_libraries(${TARGET_NAME} ${OpenVR_LINK_LIBRARIES})
    else()
        find_package(OpenVR REQUIRED)
        target_link_libraries(${TARGET_NAME} openvr::openvr)
    endif()
```
You might have noticed that our CMake files are currently a mix of 2 spaces indentation and 4 spaces indentation. We should use 4 spaces when updating stuff to be in line with our C++ coding standards.
Don't worry about it too much, but on small files like this the opportunity presents itself.